cbmackintosh/flashcards
# Feature/iteration1

**What does this PR do?**
- Completes Iteration 1
- Provides testing suites for `Card` and `Turn` classes
- Builds functional `Card` and `Turn` classes
- Adds other necessary file architecture

**Is this a feature or refactor?**
- Feature

**Problems Encountered & Resulting Solutions**
- None

**Where should the reviewer start?**
- Test suites: `test/Card-test.js` and `test/Turn-test.js`
- Classes: `src/Card.js` and `src/Turn.js`

**How should this be tested?**
- Run `npm test` from terminal to verify that all unit tests are being passed. Review unit tests to make sure they are working correctly.


**Steps To Take**
- Proceed to iteration 2. Build more unit tests for additional classes necessary to data model.
